server/sonar-server/src/main/java/org/sonar/server/organization/ws/RemoveMemberAction.java

```diff
@@ -96,6 +99,8 @@ public void handle(Request request, Response response) throws Exception {
 
       dbClient.organizationMemberDao().delete(dbSession, organizationMember.getOrganizationUuid(), organizationMember.getUserId());
       dbSession.commit();
+
+      userIndexer.index(user.getLogin());
```

Comment:
I suppose that you're doing that now because in the futur we'll add organization in the index ?
Because for the moment reindexing user is useless, no ?